x-pack/plugins/osquery/cypress/e2e/all/live_query_run.cy.ts

```diff
@@ -36,7 +32,7 @@ describe('ALL - Live Query run custom and saved', { tags: ['@ess', '@serverless'
       ecs_mapping: {},
     }).then((savedQuery) => {
       savedQueryId = savedQuery.saved_object_id;
-      savedQueryName = savedQuery.name;
+      savedQueryName = savedQuery.id;
```

Comment:
nit: are both of these `id`s?

Reply:
`name` was undefined, `id` is the name in this case